h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -136,6 +140,15 @@ protected ModelMetricsListSchemaV3 schema(int version) {
             "specifying a Compact output format will produce a per-feature contribution", values = {"Original", "Compact"}, json = false)
     public Model.Contributions.ContributionsOutputFormat predict_contributions_output_format;
 
+    @API(help = "Only for predict_contributions function - sort Shapley values and return top_n highest (optional)", json = false)
+    public int top_n;
+
+    @API(help = "Only for predict_contributions function - sort Shapley values and return top_bottom_n lowest (optional)", json = false)
+    public int top_bottom_n;
```

Comment:
top_bottom_n - that seems confusing to me, why not just bottom_n? isn't there already a naming in another our product how to call this?

Reply:
I was also struggling how to name it, bottom_n I am not aware of any similar naming in other product

Reply:
I'll rename it to `bottom_n`